nix/process-compose/cli.nix

```diff
@@ -70,6 +57,22 @@ in
         default = true;
         description = "Enable or disable the TUI for the application.";
       };
+      arguments = process-compose-flake-lib.mkProcessComposeArgumentsOption { };
+      test-arguments = process-compose-flake-lib.mkProcessComposeArgumentsOption { };
```

Comment:
Do we expect users to be able to set `test-arguments`?

Reply:
I don't know, I mainly introduced this to be able to configure the "config" argument in the same way as every other argument. And because this one depends on it being a test-process or not, I couldn't come up with another solution.

But I mean.. why not? Could be handy, no?